sql/src/test/java/io/crate/integrationtests/ViewsITest.java

```diff
@@ -77,6 +77,13 @@ public void testCreateViewFailsIfNameConflictsWithTable() {
         execute("create view t1 as select 1");
     }
 
+    @Test
+    public void testCreateViewFailsIfNameConflictsWithPartitionedTable() {
+        execute("create table t1 (x int) partitioned by (x) clustered into 1 shards with (number_of_replicas = 0)");
+        expectedException.expectMessage("Relation '" + sqlExecutor.getDefaultSchema() + ".t1' already exists");
+        execute("create view t1 as select 1");
```

Comment:
I think this tests half of the `conflictsWithView` condition. Shall we also add a test for the other condition? 
`indexMetaData.templates().containsKey(PartitionName.templateName(viewName.schema(), viewName.name())`

Reply:
This tests the new condition. There is already a test in place to check conflicts with non-partitioned tables. See https://github.com/crate/crate/pull/7098/files#diff-24897c8e99e02c40c4f50634d95916fdR73

Reply:
Ah, you are right, however, should we account for partition ident's as well?

Reply:
Instead of going for exact match on the templates have a "startsWith" match maybe? Or is that going too far?

Reply:
> Ah, you are right, however, should we account for partition ident's as well?

`CREATE VIEW` doesn't support a `PARTITION` clause, or other partition filtering semantics. Or what do you mean?

> Instead of going for exact match on the templates have a "startsWith" match maybe? Or is that going too far?

What would be the advantage of that?

Reply:
Prevent views with the name of a partition eg. prevent `schemaName..partitioned.table.12345abc`

Not sure if this is something we need to be concerned of though

Reply:
We don't allow dots in view or table names. But thanks for the pointer because I think this validation is missing right now for views =)